client/src/modules/list/components/Items/ItemsList.jsx

```diff
@@ -78,24 +76,21 @@ class ItemsList extends PureComponent {
   };
 
   render() {
-    const {
-      archived,
-      intl: { formatMessage },
-      items
-    } = this.props;
+    const { archived, items } = this.props;
     const { limit } = this.state;
 
     return (
       <Fragment>
         {!items.length && (
-          <MessageBox
-            message={
-              archived
-                ? formatMessage({ id: 'list.items-list.message-no-arch-items' })
-                : formatMessage({ id: 'list.items-list.message-no-items' })
-            }
-            type={MessageType.INFO}
-          />
+          <MessageBox type={MessageType.INFO}>
+            <FormattedMessage
+              id={
+                archived
```

Comment:
This can be extracted before the return as `const emptyMessageId = archived ? 'list.items-list.message-no-arch-items' : 'list.items-list.message-no-items';` to reduce the nesting depth.